@xen-orchestra/backups/_runners/_writers/FullRemoteWriter.js

```diff
@@ -26,11 +26,11 @@ exports.FullRemoteWriter = class FullRemoteWriter extends MixinRemoteWriter(Abst
     )
   }
 
-  async _run({ timestamp, sizeContainer, stream }) {
+  async _run({ timestamp, sizeContainer, stream, vm = this._backup.vm, vmSnapshot = this._backup.exportedVm }) {
```

Comment:
pass vm and vmSnapshot for all the call to _transfer / _run

change the constructor of the writers to need only vmUiid, config, scheduleId